### ISSUE TITLE
Detect when no arguments were passed to 'contextlib.suppress'

### ISSUE THREAD

**Issue by AIGeneratedUsername** [PyCQA/flake8-bugbear#222]
Wrong code that is not detected by Flake8:

```python
import contextlib

with contextlib.suppress():  
    raise ValueError
```
Where the Flake8 error must be raised:

```python
import contextlib

with contextlib.suppress():  # <-- ERROR: NO ARGUMENTS WERE PASSED!
    raise ValueError
```

How the correct code may look:

```python
import contextlib

with contextlib.suppress(ValueError):  
    raise ValueError
```

**Comment by cooperlees** [comment]
Hi there. This sounds like a good check for bugbear. If you would like to add it and tests I will accept this.

I did do a quick search to try see why It does not just error when no arguments are supplied and couldn't see any reason. Due to this I've opened a cpython bug to see if we can just error directly in contextlib itself as it seems a waste of a context manager construct if you don't pass anything. I can't think of a valid use case. is there one I'm missing?

Bug opened here: https://bugs.python.org/issue46819

**Comment by cooperlees** [comment]
It's been decided that nothing should be done in cpython. Let's add the check here. The cpython bug has been closed.
